app/src/main/java/app/insti/fragment/FileComplaintFragment.java

```diff
@@ -602,14 +603,29 @@ public void run() {
     }
 
     private void prepareTags() {
-        tagList = new ArrayList<>();
-        try {
-            for (int i = 0; i < TagCategories.CATEGORIES.length; i++) {
-                tagList.add(new ComplaintTag(TagCategories.CATEGORIES[i]));
+        RetrofitInterface retrofitInterface = Utils.getRetrofitInterface();
+        retrofitInterface.getTags("sessionid=" + getArguments().getString(Constants.SESSION_ID)).enqueue(new Callback<List<Venter.TagUri>>() {
+            @Override
+            public void onResponse(Call<List<Venter.TagUri>> call, Response<List<Venter.TagUri>> response)
+            {
```

Comment:
Please move the curly bracket to the line above for consistency with everything else.

Reply:
Noted